test/use_case/add_project/AddProjectInteractorUnitTest.java

```diff
@@ -0,0 +1,81 @@
+package use_case.add_project;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import entity.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class AddProjectInteractorUnitTest {
+
+    @Test
+    void successTest() {
+        AddProjectInputData inputData = new AddProjectInputData(
+                "CSC207 Group Project",
+                "Create an application, which makes use of Clean Architecture and SOLID Design principles, " +
+                        "which serves as an aid for time management and organizing deadlines. Upon adding a project, the" +
+                        "application will split the project into smaller deadlines which help the user to stay on track.",
+                "2023-12-04",
+                "foobar");
+
+        AddProjectDataAccessInterface dummyUsersDataAccessInterface = new DummyUsersDataAccessObject();
```

Comment:
SUGGESTION: change the dummyDAO creation to use the same syntax as the successPresenter (implicit class that implements the interface).